pytest_fixtures/component/oscap.py

```diff
@@ -35,11 +35,10 @@ def scap_content(import_ansible_roles, module_target_sat):
     )
     scap_id = scap_info['id']
     scap_info = entities.ScapContents(id=scap_id).read()
-
     scap_profile_id = [
         profile['id']
         for profile in scap_info.scap_content_profiles
-        if OSCAP_PROFILE[settings.oscap.profile] in profile['title']
+        if OSCAP_PROFILE[f'dsrhel{module_target_sat.os_version.major}'] in profile['title']
```

Comment:
A little suggestion here:
```suggestion
        if module_target_sat.OSCAP['dsrhel'] in profile['title']
```

see https://github.com/SatelliteQE/robottelo/blob/db5b3d5a3cffb8a827782b4ea4e979fb59c5a38a/robottelo/host_helpers/contenthost_mixins.py#L58-L64

A similar abstraction on the host level can be built for `OSCAP_DEFAULT_CONTENT`